src/web/src/utils.js

```diff
@@ -210,6 +210,22 @@ export const addDays = (date, days) => {
   return result;
 };
 
+/**
+ * Export Course Data Given CRN Value.
+ */
+export const conflictCourse = (allCourseData, inputCRNValue) => {
```

Comment:
nit: I'd suggest calling this `findCourseByCourseSessionCrn`

Reply:
Sounds good to me. I completely agree since more descriptive function names would make everything more clear. It should be updated by now.